sutta_publisher/tests/unit/test_helper_functions.py

```diff
@@ -100,20 +100,21 @@ def test_should_check_intersection_of_two_lists():
             "dn1:0.1",
             "lorem ipsum",
             "vnp1.9, pts-vp-pli14.2",
-            "<p><a class='sc-main' id='dn1:0.1'>DN 1:0.1</a><a class='pts-vp-pli' id='pts-vp-pli14.2'>PTS-VP-PLI 14.2</a>lorem ipsum",
+            "<p><span   id='dn1:0.1'></span><a class='pts-vp-pli' id='pts-vp-pli14.2'>PTS-VP-PLI 14.2</a>lorem ipsum",
         ),
         (
             "<h1 class='sutta-title'>{}</h1></header>",
             "mn138:0.2",
             "dolor sit",
             "invalid_ref, bj7.9",
-            "<h1 class='sutta-title'><a class='sc-main' id='mn138:0.2'>MN 138:0.2</a><a class='bj' id='bj7.9'>BJ 7.9</a>dolor sit</h1></header>",
+            "<h1 class='sutta-title'><span   id='mn138:0.2'></span><a class='bj' id='bj7.9'>BJ 7.9</a>dolor sit</h1></header>",
```

Comment:
The changes are fine but I wonder why we have multiple spaces between span and id. IMO there should be only 1 space. (this is unrelated to this PR)